test/FixCode/fixits-omit-return.swift

```diff
@@ -0,0 +1,14 @@
+// RUN: not %target-swift-frontend -emit-sil %s -emit-fixits-path %t.remap -diagnostics-editor-mode
+// RUN: c-arcmt-test %t.remap | arcmt-test -verify-transformed-files %s.result
+
+func ff_fixit_addreturn() -> String {
+    print("entering ff_fixit_addreturn()")
+    "foo"
```

Comment:
I think this might be better served as an `expected-error` with a FixIt range specified.

```swift
// expected-error {{missing return in a function expected to return String; did you mean to return the last expression?}} {{4-4=return }}
```

Reply:
Ah, that is much shorter!  Changed here: 
https://github.com/apple/swift/pull/23251/commits/e5947e47753a5a693ff596c3891912dde83d89b7